README.md

```diff
@@ -196,3 +173,44 @@ Note that the contents of those file will be overidden on each run.
 
 We generate a self-signed wildcard certificate to use for all the ingress
 controllers.
+
+## Development and testing
+
+To bootstrap a development environment, you need to install the
+runtime dependencies listed above, plus the development environment
+dependencies.
+
+### Running the tests
+
+This section explains how to run linters and the compliance test suites.
+
+#### Linters and formatters
+
+The codebase is checked with linters and against common formatting rules.
+
+##### Linters and formatters dependencies
+
+1. Python 3, with pip.
+1. NodeJS, with npm.
+1. See [requirements.txt](requirements.txt).
+1. See [package.json](package.json).
+
+#### Compliance test suite
+
+The test suite checks the whole environment for compliance using a verifier (
```

Comment:
The test suite is executed automatically by Travis on each commit inside a Pull Request,  according to the instructions contained in `.travis.yml`.

Reply:
Is it correct the text that I've proposed above? If so, we could add it...

Reply:
LGTM, will add

Reply:
sorry, to be more precise, I've now verified that commit runs on each commit, even outside a Pull Request:
The test suite is executed automatically by Travis on each commit, according to the instructions contained in `.travis.yml`.

Reply:
done

Reply:
it's not a big issue, but my proposal was to explicitly a sentence to explain travis, because I didn't know travis before working on this project and many people interested in kubernetes may know nothing about travis.
My proposed text was :
The test suite is executed automatically by Travis on each commit, according to the instructions contained in .travis.yml.
to be added after 
The test suite checks the whole environment for compliance using a verifier (
[InSpec](https://www.inspec.io/) in this case).

note also that you should put the new line before the ( otherwise a weird space appears in
the web page... or maybe you can avoid the space with (\

Reply:
Travis runs the whole test suite, not just the compliance one. Adding your text, but where we start talking about tests :)

Reply:
done both!